lib/container_server.go

```diff
@@ -674,11 +674,11 @@ type containerServerState struct {
 
 // AddContainer adds a container to the container state store
 func (c *ContainerServer) AddContainer(ctr *oci.Container) {
-	sandbox := c.state.sandboxes.Get(ctr.Sandbox())
-	if sandbox == nil {
+	sb := c.state.sandboxes.Get(ctr.Sandbox())
+	if sb == nil {
 		return
 	}
-	sandbox.AddContainer(ctr)
+	sb.AddContainer(ctr)
```

Comment:
I probably would have gone 'newSandbox", but shrug.

Reply:
I renamed it :)